required/latex-lab/latex-lab-firstaid.dtx

```diff
@@ -215,24 +229,17 @@
 % to make tagging more easier. 
 %    \begin{macrocode}
     \def\thmhead@plain##1##2##3{%
-      \tag_struct_begin:n{tag=Caption}  
       \thmname{##1}
       \thmnumber{
         \@ifnotempty{##1}{~}\@upn{##2}   
        }%   
-      \thmnote{\pdffakespace\space{\the\thm@notefont(##3)}}
-      %moved for easier tagging
-      \tag_mc_begin:n{}\the\thm@headpunct\tag_mc_end:  
-      \tag_struct_end:
+      \thmnote{\pdffakespace\space{\the\thm@notefont(##3)}}            
```

Comment:
Seeing this makes me wonder if one should hide the \pdffakespace\space combination ins something like \typesetspace

Reply:
I'm not sure where that could live currently. It is backend dependent and most code (for lualatex) is in tagpdf. Need some thoughts.